build_scripts/webpack-rules.js

```diff
@@ -25,9 +25,10 @@ module.exports = [
     test: /\.(js|jsx)$/,
     include: [
       path.resolve('src'),
-      path.resolve('node_modules/@neo4j/browser-lambda-parser')
+      path.resolve('node_modules/@literal-jsx/parser'),
+      path.resolve('node_modules/@neo4j/browser-lambda-parser'),
+      path.resolve('node_modules/remark-mdx')
     ],
-    // exclude: /(node_modules(?!:\/@neo4j))|(cypher-codemirror)|(test_utils)|(dist)/,
```

Comment:
I take it as this was redundant?

Reply:
It was commented out, as far as I can see, it didn't do anything

Reply:
😆 sorry, I missed the comment. Good cleanup